webapp/src/components/channel_link_label/component.tsx

```diff
@@ -13,32 +13,42 @@ interface Props {
     profiles: UserProfile[],
 }
 
+const CHANNEL_TOOLTIP_LIMIT = 10;
```

Comment:
Nice, I loved seeing this going nuts during load-tests but I suppose this is better :p 

Could we move the definition to the `constants.ts` file? Also I'd suggest to reduce the value to 8 for consistency with a couple of other things including the limit for participants in group channels. We can always sync with UX to figure out a different number if needed I suppose.